libs/designsystem/src/lib/components/slide-button/slide-button.component.scss

```diff
@@ -1,24 +1,45 @@
+@use 'sass:color';
+
+@use '~@kirbydesign/core/src/scss/interaction-state';
 @use '~@kirbydesign/core/src/scss/utils';
-@use 'slide-button.component.shared';
+
+$slide-button-container-width: 256px;
+$slide-button-spacing: 2px;
+$slide-button-width: 52px;
+$slide-button-container-height: $slide-button-width + 2 * $slide-button-spacing;
+$slide-button-container-bg-color: utils.get-color('primary');
+$slider-thumb-bg-color: utils.get-color('white');
+$slide-button-text-color: utils.get-color('primary-contrast');
+$total-slider-thumb-width: $slide-button-width + $slide-button-spacing * 2;
+$slide-button-container-radius: $slide-button-width * 0.5;
+$slide-button-text-padding: 0 $slide-button-container-radius 0 $total-slider-thumb-width;
```

Comment:
Should we remove the variables and just apply the values directly to the properties (those that are not used multiple places that is)?

Reply:
This block was moved out of https://github.com/kirbydesign/designsystem/blob/main/libs%2Fdesignsystem%2Fsrc%2Flib%2Fcomponents%2Fslide-button%2Fslide-button.component.shared.scss because it was only used in one single place. I didn't want to rewrite it in the first place, because it would look like something introduced by this PR.

It's not related to hover and interaction states so I'd prefer not to touch it unless it's very important. We should also have a look at why we're using hardcoded values here and see if there's a better solution - which is very out of scope, I think 😁